omocha-domain/src/main/java/org/omocha/domain/auction/AuctionServiceImpl.java

```diff
@@ -88,14 +87,16 @@ public AuctionInfo.RetrieveAuction retrieveAuction(AuctionCommand.RetrieveAuctio
 			.map(Image::getImagePath)
 			.collect(Collectors.toList());
 
-		// TODO : auction.getAuctionCategories를 list로 받고 처리하도록 수정해야함
-		AuctionCategory auctionCategory = auction.getAuctionCategories()
+		List<Long> categoryIds = auction.getAuctionCategories()
 			.stream()
-			.findFirst()
-			.orElseThrow(
-				() -> new CategoryNotFoundException(auction.getAuctionId(), retrieveCommand.memberId()));
+			.map(auctionCategory -> auctionCategory.getCategory().getCategoryId())
+			.collect(Collectors.toList());
```

Comment:
```suggestion
			.toList();
```
List 변환 이후 수정이 필요 없기 때문에 toList()가 더 맞는거 같아요~

Reply:
tolist로 변경했습니다!!